src/hocs/withCheckNetworkConnection/withCheckNetworkConnection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Alert } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { selectors } from 'app/state/electrumX';
+
+type checkNetworkConnectionArgs = (...args: any[]) => void;
```

Comment:
maybe instead of `any[]` use `unknown[]`?

Reply:
changed to unknown